Cannot import from `@chakra-ui/theme/components`

### Description

Since 2.4.2, we can inject themes of each component using `extendBaseTheme`.

However, when trying to import default component themes from `@chakra-ui/theme/components`, it fails with the error below.

```
Cannot find module '@chakra-ui/theme/components' or its corresponding type declarations.ts(2307)
```

It seems that `./dist/components/index.d.ts` in theme package isn't generated. Foundations has the same issue too.

### Link to Reproduction

https://codesandbox.io/s/vibrant-black-c5n2qy?file=/src/index.tsx

### Steps to reproduce

See codesandbox


### Chakra UI Version

2.4.4

### Browser

_No response_

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

_No response_

--- a/.changeset/wet-islands-look.md
+++ b/.changeset/wet-islands-look.md
@@ -1,0 +1,5 @@
+---
+"@chakra-ui/theme": patch
+---
+
+Fix issue where importing from `/components` and `/foundations` causes type errors

--- a/packages/components/theme/package.json
+++ b/packages/components/theme/package.json
@@ -51,7 +51,7 @@
     "clean-package": "2.1.1"
   },
   "scripts": {
-    "build": "tsup src/index.ts --dts",
+    "build": "tsup src/index.ts src/components/index.ts src/foundations/index.ts --dts",
     "dev": "pnpm build:fast -- --watch",
     "clean": "rimraf dist .turbo",
     "typecheck": "tsc --noEmit",

